src/routes/cards.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { cardController } = require('../controllers');
+
+const router = express.Router();
+
+router.get('/', cardController.getAll.bind(cardController));
```

Comment:
```suggestion
router.get('/', (req, res, next)=> cardController.getAll(req, res, next) );
```
so you can use arrow function heir